docs/cpp/configure-intellisense.md

```diff
@@ -0,0 +1,121 @@
+---
+Order: 13
+Area: cpp
+TOCTitle: Configure IntelliSense
+ContentId: 381b7ce1-5766-49b0-ad26-f9eedae70e63
+PageTitle: Configure IntelliSense
+DateApproved: 8/23/2023
+MetaDescription: Configure Visual Studio Code IntelliSense in the C/C++ Extension
+
+---
+# Configuring IntelliSense
```

Comment:
Prefer to avoid gerunds so "Configure IntelliSense" which also matches the TOC title.

Reply:
Switched to "Configure C/C++ IntelliSense"